dataAccess.js

```diff
@@ -0,0 +1,25 @@
+const knex = require('knex');
+const knexConfig = require('./knexfile.js');
+const db = knex(knexConfig.development);
+
+module.exports = {
+  getDishes: function() {
+    let query = db('dishes');
+
+    return query;
+  },
+  addDish: function(dish) {
+    return db('dishes').insert(dish);
+  },
+  getDish: function(id) {
+    return db('dishes').where({id});
+  },
+  getRecipes: function() {
```

Comment:
I believe there needs to be some kind of `join` so we can also get dishes associated with that recipe.